src/main/java/seedu/address/logic/commands/profile/AddProfileCommand.java

```diff
@@ -34,7 +34,8 @@ public class AddProfileCommand extends ProfileCommand {
             + PREFIX_TAG + "owesMoney";
 
     public static final String MESSAGE_SUCCESS = "New profile added: \n%1$s";
-    public static final String MESSAGE_DUPLICATE_PROFILE = "This profile already exists in the address book";
+    public static final String MESSAGE_DUPLICATE_NAME = "This name already exists in the address book";
+    public static final String MESSAGE_DUPLICATE_EMAIL = "This email already exists in the address book";
```

Comment:
Do you think it's better for this variables to be in `commons/core/Messages.java` instead, since they are reused in multiple different classes?

Reply:
Actually for this, is it better to be placed in `Messages.java` or `ProfileCommand.java` since the message is only use for profile commands?

Reply:
Hm, I think it might be better encapsulated in `ProfileCommand.java`, since it's just for profile commands.